src/lib/Components/OpaqueImageView/createGeminiUrl.tsx

```diff
@@ -0,0 +1,19 @@
+export function createGeminiUrl({
```

Comment:
Very similar functionality is scattered throughout Emission; might be worth adding a `TODO` to replace refs with this utility function, and to move this util up a level at that time.

Reply:
Can you point out an example? I'm struggling to find it.

Reply:
Looking around, this comment applies to Reaction, not Emission.